utils/thirdweb.ts

```diff
@@ -18,3 +20,40 @@ export const getIPFSAssetURI = (ipfsURI?: string) => {
 export const thirdwebClient = createThirdwebClient({
   clientId: config.thirdwebClientId,
 });
+
+export const thirdwebWallets: Record<ISupportedWalletName, Wallet> =
+  Object.fromEntries(
+    Object.entries(SUPPORTED_WALLETS).map(([walletName, walletConfig]) => {
+      if (!walletConfig.thirdwebId) {
+        return [walletName, undefined];
```

Comment:
Since this record is called thirdwebWallets, wouldn't we want to filter out prior to mapping those entries that don't have a thirdwebId?

Either that or I would think we would need to change the type of the record to 

`Record<ISupportedWalletName, Wallet | undefined>`

Reply:
you're right! cleaned in 80a0bdb3017096a67863902b65456ec8bf085c9b

Reply:
Nice